src/components/ActionsMenu/README.md

```diff
@@ -0,0 +1,25 @@
+## ActionsMenu
+An implementation of an ActionsMenu for TPAs
+
+ActionMenu is just a popup you'll get when openin 3-dors Menu, Drodpowdn etc. It does not control it's position on screen on desktop, and will always open from the bottom of the screen with full width on mobile.
+
+Child of `ActionsMenu` must be either `Divider` or `ActionsMenuItem`, see Examples section for details.
+
+
+The `ActionsMenu` implementation provides a few default overridable styles. By default site theme is used.
+
+## Theme properties (all are optional)
+| propName              | propType | description                                                |
+|-----------------------|----------|------------------------------------------------------------|
+| MenuBackgroundColor   | string   | The color of the background instead of the default color   |
+| MenuBorderWidth       | string   | The color of the menu width instead of the default one     |
```

Comment:
It should be:
`The width of the border...`

Reply:
fixed